File: lang/clojure/skeleton/test/day_11_test.clj
(ns day-11-test
  (:require
   [clojure.test :refer [deftest is testing]]
   [day-11 :refer [part1 part2]]))

(deftest day-11-test
  (testing "sample data"
    (is (= 0 (part1 "../data/11-sample.txt")))
    (is (= 0 (part2 "../data/11-sample.txt")))))

File: years/2017/elixir/test/aoc/day_11_test.exs
defmodule AoC.Day11.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert AoC.Day11.part_1(["../data/11.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert AoC.Day11.part_2(["../data/11.txt"]) == 1
    end
  end
end